h2integrate/finances/profast_base.py

```diff
@@ -713,7 +713,8 @@ def populate_profast(self, inputs):
 
             # if VarOpEx is positive, treat as a feedstock
             varopex_adjusted_tech = inputs[f"varopex_adjusted_{tech}"]
-            if np.any(varopex_adjusted_tech) > 0:
+
+            if np.any(varopex_adjusted_tech >= 0):
```

Comment:
can we just remove the check? I don't think `varopex_adjusted_tech` should ever be less than zero.

Reply:
I believe we need to retain this check. @elenya-grant chime in here if I misrepresent anything. If `VarOpEx` is negative it's treated as a co-product in PF, when otherwise it's treated as a feedstock. This leads to slightly different bookkeeping procedures for PF. really just for clarity, not due to the differences in results.